index.html

```diff
@@ -232,6 +234,32 @@ <h2>"Event Model" &mdash; Bluesky's Data Model</h2>
           <div class="col-md-5">
           </div>
         </div>
+        <hr class="featurette-divider" />
+        <div class="row">
+          <div class="col-md-7">
+            <h2>Bluesky Queue Server &mdash; Remote Control of Experiments</h2>
```

Comment:
Same as with the menu above, I think Queueserver deserves a more prominent position here---above Widgets, Suitcase, and Event Model.

Reply:
I reordered the items. May be Databroker should be moved before the Suitcase or before the Queue Server?

Reply:
I also think that Bluesky Queue Server API should not be a separate large item, but it should be linked from the description in Bluesky Queue Server item.